doc/man7/provider-base.pod

```diff
@@ -607,6 +607,78 @@ or maximum. A -1 indicates that the group should not be used in that protocol.
 
 =back
 
+=head3 "TLS-SIGALG" Capability
+
+The "TLS-SIGALG" capability can be queried by libssl to discover the list of
+TLS signature algorithms that a provider can support. Each signature supported
+can be used for client- or server-authentication in addition to the build-in
+signature algorithms.
+TLS1.3 clients can advertise the list of TLS signature algorithms they support
+in the signature_algorithms extension, and TLS servers can select an algorithm
+from the offered list that they also support. In this way a provider can add
+to the list of signature algorithms that libssl already supports with
```

Comment:
I think it would be okay to drop _with additional ones_.

Reply:
so changed in https://github.com/openssl/openssl/pull/19312/commits/e5582173f0db49a28edf0907db053ea66e10d3dc